Snippets/MsmqTransport/MsmqTransport_All/QueueCreation/QueueCreation.ps1

```diff
@@ -51,17 +51,14 @@ Function CreateQueue
     )
 
     $queuePath = '{0}\private$\{1}'-f [System.Environment]::MachineName, $queueName
-    if ([System.Messaging.MessageQueue]::Exists($queuePath))
+    if (-Not [System.Messaging.MessageQueue]::Exists($queuePath))
     {
-        $messageQueue = New-Object System.Messaging.MessageQueue($queuePath)
-        SetPermissionsForQueue $messageQueue $account
-        return
+		$messageQueue = [System.Messaging.MessageQueue]::Create($queuePath,$true)

```

Comment:
can u use spaces and not tabs. mixing tabs and spaces is not recommended in snippets https://github.com/Particular/docs.particular.net#do-not-mix-tabs-and-spaces